### DISCUSSION TITLE
Mac installation but GOG only offers Windows HoMM2 install?

### DISCUSSION THREAD

**Discussion by ronikobrosly** [ihhub/fheroes2#5814] [Q&A]
### Your question

Hello, 
I was able to install `fheroes2` on my M1 macbook pro via `homebrew` without a problem. My next step is to get the original HoMM2 game installed. The documentation recommends I install the [GOG game version](https://www.gog.com/game/heroes_of_might_and_magic_2_gold_edition) but it is only installable on Windows. How does this work? Thank you!

**Comment by oleg-derevenetz** [comment]
Hi @ronikobrosly it seems that, unfortunately, you will need to use Windows either on a VM (Windows for ARM has an x86 emulation layer) or on another x86 PC to install the original HoMM2 from GOG store.

**Comment by undef21** [comment]
@ronikobrosly you can try extract game files from GOG installer via `innoextract`.

**Comment by ihhub** [comment]
Moving this issue into Discussion section.
